src/vehicle.cpp

```diff
@@ -2624,64 +2639,114 @@ int vehicle::print_part_desc(WINDOW *win, int y1, int width, int p, int hl /*= -
     return y;
 }
 
-void vehicle::print_fuel_indicator (void *w, int y, int x, bool fullsize, bool verbose, bool desc, bool isHorizontal) const
+/**
+ * Return whether a fuel_type should be printed
+ * @param fuel_type ID of the fuel to be checked
+ * @param fullsize true if it's expected to print multiple rows
```

Comment:
How can `fullsize` be true if it's a `int`. `int` only contain numbers, if you want true/false, use a `bool`.